pkg/tinydtls/patches/0007-dtls.c-introduce-dtls_write_multi.patch

```diff
@@ -0,0 +1,82 @@
+From e61b01b45080aba95f217da9aa5bc6c7e021a401 Mon Sep 17 00:00:00 2001
```

Comment:
If we test this properly (which should be easily done with any app that includes `tinydtls`), I think we can go forward in the GitHub repo directly instead of patching.

Reply:
Yup, the patch got merged upstream now.